src/redux/books/bookSlice.js

```diff
@@ -0,0 +1,23 @@
+import { createSlice } from '@reduxjs/toolkit';
+import data from '../../components/Todo/TodoData';
+
+const initialState = {
+  bookData: data,
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to use `bookData` as an array
```suggestion
  bookData: [],
```